CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Release Date: 2021-??-?? Valhalla 3.1.5
 * **Removed**
 * **Bug Fix**
+   * FIXED: Strange detoures with multi-origin/destination unidirectional A* [#3585](https://github.com/valhalla/valhalla/pull/3585)
```

Comment:
can you put this at the bottom of the list, we have them in descending chronological order